stable_baselines3/common/atari_wrappers.py

```diff
@@ -210,32 +234,37 @@ class AtariWrapper(gym.Wrapper):
 
     Specifically:
 
-    * NoopReset: obtain initial state by taking random number of no-ops on reset.
+    * Noop reset: obtain initial state by taking random number of no-ops on reset.
     * Frame skipping: 4 by default
+    * Sticky actions: disabled by default
     * Max-pooling: most recent two observations
     * Termination signal when a life is lost.
     * Resize to a square image: 84x84 by default
     * Grayscale observation
     * Clip reward to {-1, 0, 1}
 
-    :param env: gym environment
-    :param noop_max: max number of no-ops
-    :param frame_skip: the frequency at which the agent experiences the game.
-    :param screen_size: resize Atari frame
-    :param terminal_on_life_loss: if True, then step() returns done=True whenever a life is lost.
+    :param env: Environment to wrap
+    :param noop_max: Max number of no-ops
+    :param action_repeat_probability:
+    :param frame_skip: Frequency at which the agent experiences the game.
+    :param screen_size: Resize Atari frame
+    :param terminal_on_life_loss: If True, then step() returns done=True whenever a life is lost.
     :param clip_reward: If True (default), the reward is clip to {-1, 0, 1} depending on its sign.
     """
 
     def __init__(
         self,
         env: gym.Env,
         noop_max: int = 30,
+        action_repeat_probability: float = 0.0,
         frame_skip: int = 4,
         screen_size: int = 84,
         terminal_on_life_loss: bool = True,
         clip_reward: bool = True,
     ) -> None:
-        env = NoopResetEnv(env, noop_max=noop_max)
+        if noop_max > 0:
```

Comment:
I would do the same for Sticky action

Reply:
related to https://github.com/DLR-RM/stable-baselines3/issues/271 (solved in the gym 0.26 branch)

Reply:
to avoid wrapping when not needed, and yes sampling for nothing